content/cockpit-147.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Cockpit 147
+author: Martin Pitt
+date: 2017-07-27 15:00
+tags: cockpit, linux
+slug: cockpit-147
+category: release
+summary: Cockpit with Accounts improvements
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 147.
+
+### Add configuration of expiry to Accounts page
+
+The Accounts page can now display and change the expiration of user accounts
+and their passwords:
+
+![Account Expiry](images/accounts-expiry.png)
+
+Thanks to Fridolin Pokorny for this improvement!
+
+### Consistently ignore loopback traffic on all network load graphs
+
+The network load graphics on the System, Networking, and Dashboard pages
+previously sometimes accounted traffic on the loopback network device (`lo`),
+which in most cases is not interesting for administrators. The graphs have been
+harmonized to always ignore the loopback device.
+
+### Fix the Kdump page in Internet Exporer
+
+This page previously produced errors when being opened in Internet Explorer.
```

Comment:
If you want to add more detail, you could say that the calls to unsupported JavaScript functions were replaced.